libs/ledger-live-common/src/families/internet_computer/bridge/bridgeHelpers/account.ts

```diff
@@ -98,7 +104,7 @@ export const getAccountShape: GetAccountShape = async info => {
     derivationMode,
   });
 
-  log("debug", `Generation account shape for ${address}`);
+  // log("debug", `Generation account shape for ${address}`);
```

Comment:
Minor but we could remove this if it is not needed at all

Reply:
Sorry for the late reply, I am sorry about this. I agree this can be removed and is not needed. Will not forget to remove this in future 🙌🏻